test/ClickstreamAnalytics.test.ts

```diff
@@ -69,6 +69,10 @@ describe('ClickstreamAnalytics test', () => {
 			_user_name: 'carl',
 			_user_age: 20,
 		});
+		ClickstreamAnalytics.setGlobalAttributes({
```

Comment:
Do you assert the global attributes always in the subsequent events?

Reply:
added test case for assert subsequent events contains global attributes